README.rst

```diff
@@ -109,23 +113,23 @@ tox is a prerequisite for running unit tests so you need to make sure you have i
 
 To run the integration tests, the following prerequisites must be met
 
-1. Access to an AWS account to run the tests on
-2. AWS account credentials available to boto3 clients used in the tests
-3. The AWS account has an IAM role named :code:`SageMakerRole`
-4. The libraries listed in the ``extras_require`` object in ``setup.py`` for ``test`` are installed.
-   You can do this by running the following command: :code:`pip install --upgrade .[test]`
+1. AWS account credentials are available in the environment for the boto3 client to use.
+2. The AWS account has an IAM role named :code:`SageMakerRole` with the AmazonSageMakerFullAccess policy attached.
 
-You can run integ tests by issuing the following command:
+We recommend selectively running just those integration tests you'd like to run. You can filter by individual test function names with:
 
 ::
 
-    pytest tests/integ
+    pytest -k 'test_i_care_about'
+
 
-You can also filter by individual test function names (usable with any of the previous commands):
+You can also run all of the integration tests by running the following command, but they run sequentially, so they'll take a long time to finish:
```

Comment:
maybe we should also add instructions for how to run them in parallel then? i.e. `pytest -n auto`

Reply:
Oh wow. Nice. I'll do this